requirements.txt

```diff
@@ -29,3 +29,4 @@ redis==2.9.0
 nydus==0.10.6
 Cerberus==0.5
 matplotlib
+lumberjack==git+https://github.com/CERNDocumentServer/lumberjack.git
```

Comment:
It think this should be only `git+https://github.com/CERNDocumentServer/lumberjack.git`

Once we release one version then we can also put it in the URL